app/src/gui/components/authentication/oneServerLanding.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+This is a special login landing page for the most common use case i.e. one
+listing, user not logged in.
+*/
+
+import {ListingsObject} from '@faims3/data-model/src/types';
+import LoginOutlinedIcon from '@mui/icons-material/LoginOutlined';
+import {Box, Button, Paper, Typography, useTheme} from '@mui/material';
+import {useState} from 'react';
+import {QRCodeButtonOnly, ShortCodeOnlyComponent} from './shortCodeOnly';
+import {isWeb} from '../../../utils/helpers';
+import {Browser} from '@capacitor/browser';
+import {APP_ID} from '../../../buildconfig';
+
+const OnboardingComponent = ({
+  scanQr,
+  listings,
+}: {
+  scanQr: boolean;
+  listings: ListingsObject[];
+}) => {
+  const [showCodeInput, setShowCodeInput] = useState(false);
+  const theme = useTheme();
+
+  // This component is only rendered when this item is defined
+  const listing = listings[0]!;
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        // TODO I don't like this magic number but can't workout how app bar
+        // height is calculated and is not screen size dependent
+
+        // Adjust for header height to achieve true center
+        height: 'calc(100vh - 110px)',
+        padding: 1,
+        backgroundColor: '#f5f5f5',
+      }}
+    >
+      <Paper
+        elevation={2}
+        sx={{
+          width: '100%',
+          maxWidth: 380,
+          padding: '32px 24px',
+          borderRadius: '28px',
+          display: 'flex',
+          flexDirection: 'column',
+          gap: 3,
+          backgroundColor: '#ffffff',
+        }}
+      >
+        <Typography
+          variant="h3"
+          component="h1"
+          sx={{
+            textAlign: 'center',
+            fontWeight: 500,
+            color: theme.palette.primary.dark,
+            marginBottom: 1,
+          }}
+        >
+          Welcome
+        </Typography>
+
+        {/* Sign In Button */}
+        <Button
+          variant="outlined"
+          fullWidth
+          startIcon={
+            <LoginOutlinedIcon sx={{color: theme.palette.primary.main}} />
+          }
+          onClick={async () => {
```

Comment:
Can you refactor this into a function that can be used in both places rather than copying here.

Reply:
scratch that, will do it in a later PR